config/maintainers.json

```diff
@@ -80,6 +80,16 @@
       "link_url": null,
       "avatar_url": "https://s.gravatar.com/avatar/8fd910b4f7c7baabc22976dc6b29b744",
       "bio": "Elixir developer based in Los Angeles, former instructor at USC"
+    },
+    {
+      "github_username": "angelikatyborska",
+      "alumnus": false,
+      "show_on_website": false,
```

Comment:
I actually like putting my name on nice things, but I saw that on https://github.com/exercism/elixir/pull/529 fireproofsocks was asked to change theirs to `false`, so I took that as a hint that it's not appropriate to show up on the v2 track if what I am doing is working on v3.

Reply:
I think that was done with many of the new maintainers because they were brought in on the wave following that initial call for maintainers from the mentors because many of them were relatively new and unknown. 

I think there is a good case for yours to be set to true because you have been on exercism as a mentor for a while, have previously contributed to this track. 

I would support it if there is agreement to it. @ihid / @ erikshierboom?